### PR TITLE
[Improve][Document] Update the guide and add example

### PR DESCRIPTION
**Pull request by yujian225**
Update [deployment.md]
Update [quick-start-seatunnel-engine.md] Added an extended example （mysql to doris）

### CODE REVIEW

**Comment by Hisoka-X** [comment]
Please update english doc too. Thanks.

**Comment by yujian225** [comment]
> 请也更新英文文档。谢谢。

已更新

File: docs/en/start-v2/locally/deployment.md [hunk 1]
```diff
@@ -28,27 +28,13 @@ tar -xzvf "apache-seatunnel-${version}-bin.tar.gz"
...
-Usually you don't need all connector plugins, so you can specify the plugins you need through configuring `config/plugin_config`. For example, if you only need the `connector-console` plugin, you can modify the plugin.properties configuration file as follows:
-
-```plugin_config
---seatunnel-connectors--
-connector-console
---end--
-```
-
-If you want the example application to work properly, you need to add the following plugins.
+Typically, you do not need all the connector plugins. You can specify the required plugins by configuring `config/plugin_config`. For example, if you want the sample application to work properly, you will need the `connector-console` and `connector-fake` plugins. You can modify the `plugin.properties` configuration file as follows:
```
Comment (Hisoka-X) [inline docs/en/start-v2/locally/deployment.md]:
```suggestion
Typically, you do not need all the connector plugins. You can specify the required plugins by configuring `config/plugin_config`. For example, if you want the sample application to work properly, you will need the `connector-console` and `connector-fake` plugins. You can modify the `plugin_config` configuration file as follows:
```

File: docs/en/start-v2/locally/deployment.md [hunk 2]
```diff
@@ -28,27 +28,13 @@ tar -xzvf "apache-seatunnel-${version}-bin.tar.gz"
...
 
 ```bash
 sh bin/install-plugin.sh
 ```
 
-If you need a specific connector version, taking 2.3.7 as an example, you need to execute the following command:
-
-```bash
-sh bin/install-plugin.sh 2.3.7
-```
```
Comment (Hisoka-X) [inline docs/en/start-v2/locally/deployment.md]: why delete this part?

Comment (yujian225) [reply docs/en/start-v2/locally/deployment.md]: The issue mentioned above has been resolved.

File: docs/en/start-v2/locally/deployment.md (hunk 1 above)
Comment (yujian225) [reply docs/en/start-v2/locally/deployment.md]: The issue mentioned above has been resolved.

**Comment by Hisoka-X** [comment]
Please follow the guide to open github action on your fork repository. Thanks 
image

File: docs/en/start-v2/locally/quick-start-seatunnel-engine.md [hunk 3]
```diff
@@ -95,7 +95,103 @@ The SeaTunnel console will print some logs as below:
...
+
+## Step 4: Run the SeaTunnel Application
+
+You can start the application using the following command:
+
+:::tip
+
+Starting from version 2.3.1, the `-e` parameter in `seatunnel.sh` has been deprecated. Please use the `-m` parameter instead.
+
+:::
```
Comment (Hisoka-X) [inline docs/en/start-v2/locally/quick-start-seatunnel-engine.md]:
```suggestion
```

File: docs/zh/start-v2/locally/quick-start-seatunnel-engine.md [hunk 4]
```diff
@@ -94,7 +94,102 @@ SeaTunnel控制台将会打印一些如下日志信息:
...
+
+## 步骤 4: 运行SeaTunnel应用程序
+
+您可以通过以下命令启动应用程序：
+
+:::tip
+
+从2.3.1版本开始，seatunnel.sh中的-e参数被废弃，请改用-m参数。
+
+:::
```
Comment (Hisoka-X) [inline docs/zh/start-v2/locally/quick-start-seatunnel-engine.md]:
```suggestion
```

**Review by Hisoka-X — APPROVED** [review]
Thanks @yujian225

**Review by hailin0 — APPROVED** [review]

**Pull request merged**